substrate/frame/staking/src/pallet/mod.rs

```diff
@@ -89,13 +93,27 @@ pub mod pallet {
 
 	#[pallet::config(with_default)]
 	pub trait Config: frame_system::Config {
+		/// The old trait for staking balance. Deprecated and only used for migrating old ledgers.
```

Comment:
Shouldn't the pallet be annotated with deprecated then?

Reply:
Not pallet but yeah may be config item can be annotated as deprecated (if its possible).